.github/actions/lockfiles/action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Lockfile'
+description: 'Forbid non-npm lockfiles'
+runs:
+    using: composite
+    steps:
+        - name: Check lockfiles
+          uses: actions/github-script@v6
+          with:          
+            script: |
+              const {existsSync} = require('fs');
+              ['yarn.lock', 'pnpm-lock.yaml'].forEach(forbiddenLockFile => {
+                if(existsSync(forbiddenLockFile)) {
+                  const error = `Lockfile ${forbiddenLockFile} is not allowed in this repo, remove it.`;
+                  core.error(
```

Comment:
not sure where core is coming from

Reply:
From https://github.com/actions/github-script